<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Basic Usage Example for QtNoidApp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Application or Application Bundle path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtAppBundle"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Config.json file path using filePathAsAppSibling</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtConfigPath"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>groupNameFromObjectOrClass</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="txtGroupName"/>
      </item>
      <item>
       <widget class="QPushButton" name="cmdGroupName">
        <property name="text">
         <string>Set MainWindow Name to AppTest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cmdFullDialogGrab">
        <property name="text">
         <string>fullDialogGrab</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QRadioButton" name="optUpdateWindowTitle">
      <property name="text">
       <string>UpdateWindowsTitle (Changed* / Unchanged)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>FullDialogGrab Result</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="txtFullDialogGrab">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>388</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
